
CondaSystemExit: Exiting.

WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
    WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
    WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)


==> WARNING: A newer version of conda exists. <==
  current version: 4.14.0
  latest version: 23.5.0

Please update conda by running

    $ conda update -n base -c defaults conda


WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
    WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
    WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
torch 2.0.1 requires nvidia-cudnn-cu11==8.5.0.96; platform_system == "Linux" and platform_machine == "x86_64", but you have nvidia-cudnn-cu11 8.6.0.163 which is incompatible.
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution -vidia-cudnn-cu11 (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
WARNING: Ignoring invalid distribution - (/usr/local/Miniconda3-py39_4.10.3-Linux-x86_64/envs/ssci/lib/python3.10/site-packages)
2023-05-31 12:29:54.926607: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-05-31 12:29:55.635722: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2023-05-31 12:29:58.554529: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-05-31 12:29:59.429941: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/machmeier/network-traffic-classification/train_vit.py", line 16, in <module>
    from new_model import ViTClassifier, get_config
  File "/home/machmeier/network-traffic-classification/new_model.py", line 1, in <module>
    import ml_collections
ModuleNotFoundError: No module named 'ml_collections'
